src/runner/browser-env/vite/browser-modules/driver.ts

```diff
@@ -67,25 +73,23 @@ function getAllProtocolCommands(): string[] {
 }
 
 function getMockedProtocolCommands(): PropertiesObject {
-    return [...getAllProtocolCommands(), ...SERVER_HANDLED_COMMANDS, ...window.__testplane__.customCommands].reduce(
-        (acc, commandName) => {
-            acc[commandName] = { value: mockCommand(commandName) };
-            return acc;
-        },
-        {} as PropertiesObject,
-    );
+    return [...getAllProtocolCommands(), ...SERVER_HANDLED_COMMANDS].reduce((acc, commandName) => {
+        acc[commandName] = { value: mockCommand(commandName) };
+        return acc;
+    }, {} as PropertiesObject);
 }
 
 function mockCommand(commandName: string): ProtocolCommandFn {
-    return async (...args: unknown[]): Promise<unknown> => {
+    return async function (this: WebdriverIO.Browser | WebdriverIO.Element, ...args: unknown[]): Promise<unknown> {
         const { socket } = window.__testplane__;
         const timeout = getCommandTimeout(commandName);
+        const element = isWdioElement(this) ? this : undefined;
```

Comment:
Using `undefined` is considered a bad practice.

I suggest to get rid of it. For example, we could use "null" or:
```ts
const opts = { name: commandName, args };

if (isWdioElement(this)) {
    opts.element = this;
}
```

Reply:
I don't see a real problem with using `undefined` here. It just gives me ability to write code shorter, but result the same as in your example.